test/bats/tests/config/config_v1beta1_verifier_notation_crl.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: Verifier
+metadata:
+  name: verifier-notation
+spec:
+  name: notation
+  artifactTypes: application/vnd.cncf.notary.signature
+  parameters:
+    verificationCertStores:
+      ca:
+        ca-crl:
+          - keymanagementprovider-inline
+    trustPolicyDoc:
+      version: "1.0"
+      trustPolicies:
+        - name: default
+          registryScopes:
+            - "*"
+          signatureVerification:
+            level: audit
+          trustStores:
+            - ca:ca-crl
```

Comment:
if the underlying kmp is the same, probably we don't need to rename the trust store name.

Reply:
This is kmp contains the CRL root cert. And it is different from the default ones.